context/app/static/js/pages/Organ/hooks.ts

```diff
@@ -159,25 +159,25 @@ export function useLabelledDatasetsQuery(searchItems: string[]) {
   return datasetsHits.map((hit) => hit._id);
 }
 
-export function useDataProducts(organName: string) {
+export function useDataProducts(organ: OrganFile) {
+  const { name, search } = organ;
   const { dataProductsEndpoint } = useAppContext();
-  const dataProductsUrl = `${dataProductsEndpoint}/api/data_products/${organName}`;
-
-  const { data, isLoading } = useSWR<OrganDataProducts[]>(
-    dataProductsUrl,
-    (url: string) =>
-      fetcher({
-        url,
-      }),
-    { fallbackData: [] },
-  );
 
-  const dataProducts = data ?? [];
+  const isLateral = search.length > 1;
+
+  const possibleNames = isLateral ? [`${name} (Right)`, `${name} (Left)`] : [name];
+  const urls = possibleNames.map((n) => `${dataProductsEndpoint}/api/data_products/${n}`);
 
-  const dataProductsWithUUIDs = dataProducts.map((product) => {
-    const datasetUUIDs = product.dataSets.map((dataset) => dataset.uuid);
-    return { ...product, datasetUUIDs };
+  const { data, isLoading } = useSWR<OrganDataProducts[]>(urls, (u: string[]) => multiFetcher({ urls: u }), {
+    fallbackData: [],
   });
 
+  const dataProducts = (data ?? []).filter(Boolean).flat();
```

Comment:
```suggestion
  const dataProducts = (data ?? []).filter(Boolean).flat();
```
What is the `filter(Boolean)` doing here?